zerver/lib/actions.py

```diff
@@ -2856,8 +2856,8 @@ def decode_email_address(email):
 # the code pretty ugly, but in this case, it has significant
 # performance impact for loading / for users with large numbers of
 # subscriptions, so it's worth optimizing.
-def gather_subscriptions_helper(user_profile):
-    # type: (UserProfile) -> Tuple[List[Dict[str, Any]], List[Dict[str, Any]], List[Dict[str, Any]]]
+def gather_subscriptions_helper(user_profile, include_subscribers=True):
```

Comment:
It makes me really nervous to call this already complex function for the mobile case, since we're basically just ignoring all the peer subscribers.  This is a case where I'd much rather have two mostly similar functions where the second function is dirt simple and is optimized for both readability and efficiency in the don't-care-about-peers case.